integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/mediatorconfig/call_blocking/AcceptedEndpointProxy.xml

```diff
@@ -0,0 +1,36 @@
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+<proxy xmlns="http://ws.apache.org/ns/synapse"
+       name="AcceptedEndpointProxy"
+       startOnLoad="true"
+       statistics="disable"
+       trace="disable"
+       transports="http,https">
+    <target>
+        <inSequence>
+            <log level="full"/>
+            <property action="remove" name="NO_ENTITY_BODY" scope="axis2"/>
+            <property name="ContentType" scope="axis2" value="text/plain"/>
+            <property name="messageType" scope="axis2" value="text/plain"/>
```

Comment:
Please check the possibility of getting this work without these properties

Reply:
I tried. But we need these lines to get the response